Gemfile.lock

```diff
@@ -41,5 +43,8 @@ DEPENDENCIES
   sinatra
   test-unit
 
+RUBY VERSION
+   ruby 2.3.0p0
+
```

Comment:
I think that's something new bundler started doing to make sure the ruby version is captured in the Gemfile.lock.